### ISSUE TITLE
[Bug]: FTB One chunk crashing server

### ISSUE THREAD

**Issue by wqwq29** [FTBTeam/FTB-Modpack-Issues#1156]
### Modpack

FTB One

### Modpack version

1.3

### Log Files

```
---- Error seen when loading up server ----

[main/ERROR] [ne.mi.co.tr.CoreModBaseTransformer/COREMOD]: Error occurred apply
ing transform of coremod coremods/field_to_method.js function biome

---- Minecraft Crash Report ----
// Don't be sad. I'll do better next time, I promise!

Time: 7/23/22, 8:27 PM
Description: Watching Server

java.lang.Error: ServerHangWatchdog detected that a single server tick took 60.00 seconds>
at com.google.common.collect.Maps.newEnumMap(Maps.java:423) ~[guava-31.0.1-jre.ja>
at net.minecraft.world.level.material.FlowingFluid.m_76079_(FlowingFluid.java:332>
at net.minecraft.world.level.material.FlowingFluid.m_76014_(FlowingFluid.java:137>
at net.minecraft.world.level.material.FlowingFluid.m_76010_(FlowingFluid.java:124>
at net.minecraft.world.level.material.FlowingFluid.m_6292_(FlowingFluid.java:415)>
at net.minecraft.world.level.material.FluidState.m_76163_(FluidState.java:76) ~[s>
at net.minecraft.world.level.chunk.LevelChunk.m_62812_(LevelChunk.java:490) ~[ser>
at net.minecraft.server.level.ChunkMap.m_214959_(ChunkMap.java:722) ~[server-1.19>
at net.minecraft.server.level.ChunkMap$$Lambda$19024/0x0000000802cddbb0.accept(Un>
at com.mojang.datafixers.util.Either$Left.ifLeft(Either.java:43) ~[datafixerupper>
at net.minecraft.server.level.ChunkMap.m_214929_(ChunkMap.java:721) ~[server-1.19>
at net.minecraft.server.level.ChunkMap$$Lambda$18863/0x0000000802cb2990.apply(Unk>
at java.util.concurrent.CompletableFuture$UniApply.tryFire(Unknown Source) ~[?:?]>
at java.util.concurrent.CompletableFuture$Completion.run(Unknown Source) ~[?:?] {}
at net.minecraft.util.thread.BlockableEventLoop.m_6367_(BlockableEventLoop.java:1>
at net.minecraft.server.level.ServerChunkCache$MainThreadExecutor.m_6367_(ServerC>
at net.minecraft.util.thread.BlockableEventLoop.m_7245_(BlockableEventLoop.java:1>
at net.minecraft.server.level.ServerChunkCache$MainThreadExecutor.m_7245_(ServerC>
at net.minecraft.server.level.ServerChunkCache.m_8466_(ServerChunkCache.java:267)>
at net.minecraft.server.MinecraftServer.m_129961_(MinecraftServer.java:751) ~[ser>
at net.minecraft.server.MinecraftServer.m_7245_(MinecraftServer.java:740) ~[serve>
at net.minecraft.util.thread.BlockableEventLoop.m_18699_(BlockableEventLoop.java:>
at net.minecraft.server.MinecraftServer.m_130012_(MinecraftServer.java:725) ~[ser
at net.minecraft.server.MinecraftServer.m_130011_(MinecraftServer.java:658) ~[ser>
at net.minecraft.server.MinecraftServer.m_177918_(MinecraftServer.java:245) ~[ser>
at net.minecraft.server.MinecraftServer$$Lambda$18158/0x0000000802b508d8.run(Unkn>
at java.lang.Thread.run(Unknown Source) [?:?] {}


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Server Watchdog
Stacktrace:
at com.google.common.collect.Maps.newEnumMap(Maps.java:423) ~[guava-31.0.1-jre.ja>
at net.minecraft.world.level.material.FlowingFluid.m_76079_(FlowingFluid.java:332>
at net.minecraft.world.level.material.FlowingFluid.m_76014_(FlowingFluid.java:137>
at net.minecraft.world.level.material.FlowingFluid.m_76010_(FlowingFluid.java:124>
at net.minecraft.world.level.material.FlowingFluid.m_6292_(FlowingFluid.java:415)>
at net.minecraft.world.level.material.FluidState.m_76163_(FluidState.java:76) ~[s>
at net.minecraft.world.level.chunk.LevelChunk.m_62812_(LevelChunk.java:490) ~[ser>
at net.minecraft.server.level.ChunkMap.m_214959_(ChunkMap.java:722) ~[server-1.19>
at net.minecraft.server.level.ChunkMap$$Lambda$19024/0x0000000802cddbb0.accept(Un>
at com.mojang.datafixers.util.Either$Left.ifLeft(Either.java:43) ~[datafixerupper>
at net.minecraft.server.level.ChunkMap.m_214929_(ChunkMap.java:721) ~[server-1.19>
at net.minecraft.server.level.ChunkMap$$Lambda$18863/0x0000000802cb2990.apply(Unk>
at java.util.concurrent.CompletableFuture$UniApply.tryFire(Unknown Source) ~[?:?]>
at java.util.concurrent.CompletableFuture$Completion.run(Unknown Source) ~[?:?] {}
at net.minecraft.util.thread.BlockableEventLoop.m_6367_(BlockableEventLoop.java:1>
at net.minecraft.server.level.ServerChunkCache$MainThreadExecutor.m_6367_(ServerC>
-- Thread Dump --
Details:
Threads: "Reference Handler" daemon prio=10 Id=2 RUNNABLE
at java.base@17.0.2/java.lang.ref.Reference.waitForReferencePendingList(Native Me>
at java.base@17.0.2/java.lang.ref.Reference.processPendingReferences(Unknown Sour>
at java.base@17.0.2/java.lang.ref.Reference$ReferenceHandler.run(Unknown Source)


"Finalizer" daemon prio=8 Id=3 WAITING on java.lang.ref.ReferenceQueue$Lock@67d7a057
at java.base@17.0.2/java.lang.Object.wait(Native Method)
-  waiting on java.lang.ref.ReferenceQueue$Lock@67d7a057
at java.base@17.0.2/java.lang.ref.ReferenceQueue.remove(Unknown Source)
at java.base@17.0.2/java.lang.ref.ReferenceQueue.remove(Unknown Source)
at java.base@17.0.2/java.lang.ref.Finalizer$FinalizerThread.run(Unknown Source)


"Signal Dispatcher" daemon prio=9 Id=4 RUNNABLE


"Common-Cleaner" daemon prio=8 Id=10 TIMED_WAITING on java.lang.ref.ReferenceQueue$Lock@6
at java.base@17.0.2/java.lang.Object.wait(Native Method)
-  waiting on java.lang.ref.ReferenceQueue$Lock@691052dd
at java.base@17.0.2/java.lang.ref.ReferenceQueue.remove(Unknown Source)
at java.base@17.0.2/jdk.internal.ref.CleanerImpl.run(Unknown Source)
at java.base@17.0.2/java.lang.Thread.run(Unknown Source)
at java.base@17.0.2/jdk.internal.misc.InnocuousThread.run(Unknown Source)


"Notification Thread" daemon prio=9 Id=11 RUNNABLE


"Thread-0" daemon prio=5 Id=19 TIMED_WAITING
at java.base@17.0.2/jdk.internal.misc.Unsafe.park(Native Method)
at java.base@17.0.2/java.util.concurrent.locks.LockSupport.parkNanos(Unknown Sour>
at MC-BOOTSTRAP/com.electronwill.nightconfig.core@3.6.4/com.electronwill.nightcon>


"FileSystemWatchService" daemon prio=5 Id=20 RUNNABLE (in native)
at java.base@17.0.2/sun.nio.fs.LinuxWatchService.poll(Native Method)
at java.base@17.0.2/sun.nio.fs.LinuxWatchService$Poller.run(Unknown Source)
at java.base@17.0.2/java.lang.Thread.run(Unknown Source)


"Timer hack thread" daemon prio=5 Id=22 TIMED_WAITING
at java.base@17.0.2/java.lang.Thread.sleep(Native Method)
at TRANSFORMER/minecraft@1.19/net.minecraft.Util$9.run(Util.java:667)


"FileSystemWatchService" daemon prio=5 Id=26 RUNNABLE (in native)
at java.base@17.0.2/sun.nio.fs.LinuxWatchService.poll(Native Method)
at java.base@17.0.2/sun.nio.fs.LinuxWatchService.poll(Native Method)
at java.base@17.0.2/sun.nio.fs.LinuxWatchService$Poller.run(Unknown Source)
at java.base@17.0.2/java.lang.Thread.run(Unknown Source)


"Config Lib config file watcher" daemon prio=5 Id=27 WAITING on java.util.concurrent.lock>
at java.base@17.0.2/jdk.internal.misc.Unsafe.park(Native Method)
-  waiting on java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObje>
at java.base@17.0.2/java.util.concurrent.locks.LockSupport.park(Unknown Source)
at java.base@17.0.2/java.util.concurrent.locks.AbstractQueuedSynchronizer$Conditi>
at java.base@17.0.2/java.util.concurrent.ForkJoinPool.unmanagedBlock(Unknown Sour>
at java.base@17.0.2/java.util.concurrent.ForkJoinPool.managedBlock(Unknown Source)
at java.base@17.0.2/java.util.concurrent.locks.AbstractQueuedSynchronizer$Conditi>
at java.base@17.0.2/java.util.concurrent.LinkedBlockingDeque.takeFirst(Unknown So>
at java.base@17.0.2/java.util.concurrent.LinkedBlockingDeque.take(Unknown Source)
...


"Thread-2" daemon prio=5 Id=28 TIMED_WAITING
at java.base@17.0.2/jdk.internal.misc.Unsafe.park(Native Method)
at java.base@17.0.2/java.util.concurrent.locks.LockSupport.parkNanos(Unknown Sour>
at TRANSFORMER/mobcapturingtool@2.0.1/com.matyrobbrt.mobcapturingtool.util.FileWa>


"FileSystemWatchService" daemon prio=5 Id=29 RUNNABLE (in native)
at java.base@17.0.2/sun.nio.fs.LinuxWatchService.poll(Native Method)
at java.base@17.0.2/sun.nio.fs.LinuxWatchService$Poller.run(Unknown Source)
at java.base@17.0.2/java.lang.Thread.run(Unknown Source)

"FTB Backups Config Watcher 0" prio=5 Id=30 WAITING on java.util.concurrent.locks.Abstrac>
at java.base@17.0.2/jdk.internal.misc.Unsafe.park(Native Method)
-  waiting on java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObje>
at java.base@17.0.2/java.util.concurrent.locks.LockSupport.park(Unknown Source)
at java.base@17.0.2/java.util.concurrent.locks.AbstractQueuedSynchronizer$Conditi>
at java.base@17.0.2/java.util.concurrent.ForkJoinPool.unmanagedBlock(Unknown Sour>
at java.base@17.0.2/java.util.concurrent.ForkJoinPool.managedBlock(Unknown Source)
at java.base@17.0.2/java.util.concurrent.locks.AbstractQueuedSynchronizer$Conditi>
at java.base@17.0.2/java.util.concurrent.LinkedBlockingDeque.takeFirst(Unknown So>
at java.base@17.0.2/java.util.concurrent.LinkedBlockingDeque.take(Unknown Source)
...

Number of locked synchronizers = 1
- java.util.concurrent.ThreadPoolExecutor$Worker@73725278


"FileSystemWatchService" daemon prio=5 Id=31 RUNNABLE (in native)
at java.base@17.0.2/sun.nio.fs.LinuxWatchService.poll(Native Method)
at java.base@17.0.2/sun.nio.fs.LinuxWatchService$Poller.run(Unknown Source)
at java.base@17.0.2/java.lang.Thread.run(Unknown Source)


"KubeJS Background Thread" daemon prio=5 Id=33 TIMED_WAITING
at java.base@17.0.2/java.lang.Thread.sleep(Native Method)
at TRANSFORMER/kubejs@1900.5.5-build.27/dev.latvian.mods.kubejs.util.KubeJSBackgr>


"pool-7-thread-1" prio=5 Id=34 WAITING on java.util.concurrent.locks.AbstractQueuedSynchr>
at java.base@17.0.2/jdk.internal.misc.Unsafe.park(Native Method)
```

### Describe the bug

Whenever a player in a certain chunk would load in, would cause his client to crash and then eventually the server to crash.

Server has been overloaded due to unknown reasons as well.

Problem was solved mostly by tping the player out of area.

### Steps to reproduce

1. One specific player connects to server

### Expected behaviour

The server loading in the chunk

### Additional information

No need for help to solve, mainly wanted to submit for reference.

Server run on AWS Lightsail with 8Gb RAM

**Comment by wqwq29** [comment]
at java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateSequential(Unknown Source) 
~[?:?] {}
at java.util.stream.AbstractPipeline.evaluate(Unknown Source) ~[?:?] {}
at java.util.stream.ReferencePipeline.forEach(Unknown Source) ~[?:?] {}
at net.minecraft.world.level.levelgen.placement.PlacedFeature.m_226368_(PlacedFeat
ure.java:55) ~[server-1.19-20220607.102129-srg.jar%23258!/:?] {re:classloading,pl:epic_rei
_lmao:B}
at net.minecraft.world.level.levelgen.placement.PlacedFeature.m_226377_(PlacedFeat
ure.java:44) ~[server-1.19-20220607.102129-srg.jar%23258!/:?] {re:classloading,pl:epic_rei
_lmao:B}
at net.minecraft.world.level.chunk.ChunkGenerator.m_213609_(ChunkGenerator.java:46
4) ~[server-1.19-20220607.102129-srg.jar%23258!/:?] {re:classloading,pl:epic_rei_lmao:B,pl
:accesstransformer:B,re:mixin,pl:accesstransformer:B}
at net.minecraft.world.level.chunk.ChunkStatus.m_223313_(ChunkStatus.java:145) ~[s
erver-1.19-20220607.102129-srg.jar%23258!/:?] {re:classloading,pl:epic_rei_lmao:B,pl:acces
stransformer:B}
at net.minecraft.world.level.chunk.ChunkStatus.m_223279_(ChunkStatus.java:292) ~[s
erver-1.19-20220607.102129-srg.jar%23258!/:?] {re:classloading,pl:epic_rei_lmao:B,pl:acces
stransformer:B}
at net.minecraft.server.level.ChunkMap.lambda$scheduleChunkGeneration$25(ChunkMap.
java:619) ~[server-1.19-20220607.102129-srg.jar%23258!/:?] {re:mixin,pl:accesstransformer:
B,re:classloading,pl:epic_rei_lmao:B,pl:accesstransformer:B,pl:mixin:APP:polylib.mixins.js
on:MixinChunkMap,pl:mixin:APP:ftbchunks-common.mixins.json:ChunkMapMixin,pl:mixin:APP:ars_
nouveau.mixins.json:camera.ChunkMapMixin,pl:mixin:A}
at com.mojang.datafixers.util.Either$Left.map(Either.java:38) ~[datafixerupper-5.0
.28.jar%2373!/:?] {}
at net.minecraft.server.level.ChunkMap.lambda$scheduleChunkGeneration$27(ChunkMap.
java:617) ~[server-1.19-20220607.102129-srg.jar%23258!/:?] {re:mixin,pl:accesstransformer:
B,re:classloading,pl:epic_rei_lmao:B,pl:accesstransformer:B,pl:mixin:APP:polylib.mixins.js
on:MixinChunkMap,pl:mixin:APP:ftbchunks-common.mixins.json:ChunkMapMixin,pl:mixin:APP:ars_
nouveau.mixins.json:camera.ChunkMapMixin,pl:mixin:A}
... 12 more

Another error

**Comment by Saereth** [comment]
I know its been a while but if you anyone gets this again, set your watchdog timeout to -1 so you get a proper crash log and open a new issue. I've not seen this particular issues anymore in our aux data so closing for now
